src/core/Test_stat.ml

```diff
@@ -2,16 +2,65 @@ open Common
 open Test
 module PB = PrintBox
 
+(* Aggregate function computing the mean and the standard deviation
+   of data series using the Welford's algoritm. *)
+module Stats = struct
+  type acc = { n: int; total: float; mean: float * float; s: float * float }
```

Comment:
you probably need to install ocamlformat 0.24.1 and format the new code!

Reply:
I did with the right version of `ocamlformat`.